### DISCUSSION TITLE
Is it possible to have Android app have drift encryption same applicaion in ios with out encryption

### DISCUSSION THREAD

**Discussion by msarkrish** [simolus3/drift#2442] [Q&A]
I need to have encryption option for android part and for ios part no need encryption. Is it possible to implement this with Drift. 
Do you have any idea @simolus3

**Comment by simolus3** [comment]
Sure! You can use the [sqlcipher_flutter_libs](https://pub.dev/packages/sqlcipher_flutter_libs) package for both platforms instead of `sqlite3_flutter_libs` (but be sure to read its readme for iOS to avoid incompatibilities).

With that package, you need to encrypt the database explicitly by running a [`pragma key`](https://www.zetetic.net/sqlcipher/sqlcipher-api/#PRAGMA_key) command as described [here](https://drift.simonbinder.eu/docs/other-engines/encryption/#encrypted-version-of-a-nativedatabase). You can simply wrap that pragma in an if so that it only runs on Android. In that case, the iOS version would use a regular sqlite3 database.
